Source/items.cpp

```diff
@@ -1371,6 +1423,7 @@ void GetItemAttrs(int i, int idata, int lvl)
 	item[i]._iPrePower = -1;
 	item[i]._iSufPower = -1;
 
+#ifndef HELLFIRE
 	if (AllItemsList[idata].iMiscId == IMISC_BOOK)

```

Comment:
Should this actually have been `if (item[i]._iMiscId == IMISC_BOOK)`?

Reply:
It's different code in hellfire and in original, it's observable in asm

Reply:
When using `item[i]` it produces the same asm for me in 1.09. Thats also how it was written in the beta.